src/commandline/flags.md

```diff
@@ -1,4 +1,4 @@
-## Flags
+## Flags {#flags}
```

Comment:
Uh?

Reply:
I wanted to have internal link from r2yara to flags section. But it seems it is not the correct syntax. Do you know it for gitbook ?

Reply:
I don't think this would be needed, since `## Flags` already set the anchor `flags` without any header attribute `{}`.

But I think it would be a nice idea if you want to specify a different anchor name like `r2yara-flags` or similar. Since in a full document it can be more than one `Flags` title and the initial one would be ambiguous.

In any case, both [pandoc](https://pandoc.org/MANUAL.html#extension-header_attributes) and [mdbook](https://rust-lang.github.io/mdBook/format/markdown.html#heading-attributes) support this syntax.